source/partials/_developers-first.haml

```diff
@@ -7,6 +7,14 @@
         Aptible helps secure the internet by giving developers the tools and
         expertise to protect sensitive data.
     .guides-intro__languages
-      - data.quickstart.each do |title, language|
-        %a.guides-intro__language{ href: quickstart_index_href(language), class: language.slug }
-          = partial "images/icons/#{language.svg}"
+      %a.guides-intro__language.python{ href: "#{base_doc_url}/documentation/enclave/tutorials/quickstart-guides/python/django.html" }
+        = partial 'images/icons/python.svg'
```

Comment:
Do you think we should link to the language page for all of these here?

(i.e. link the "Python" page rather than the "Django" page, etc?)